thunder.install

```diff
@@ -311,9 +312,51 @@ function thunder_update_8102() {
  */
 function thunder_update_8103() {
 
-  \Drupal::service('module_installer')->install([
-    'thunder_updater',
-  ]);
+  /** @var \Drupal\thunder\ThunderUpdateLogger $updateLogger */
+  $updateLogger = Drupal::service('logger.thunder_update');
+  $updateLogger->cleanLogs();
+
+  $moduleInstaller = \Drupal::service('module_installer');
+
+  try {
+    if (!$moduleInstaller->install(['thunder_updater'])) {
+      $updateLogger->warning(t('Unable to enable the Thunder Updater'));
+    }
+  }
+  catch (MissingDependencyException $e) {
+    $updateLogger->warning(t('Unable to enable the Thunder Updater because of missing dependencies.'));
```

Comment:
If thunder updater module is not installed, then we can't use service provided by it, that means that rest of execution is not possible -> so we should simply just return from update hook.